src/ws/cockpitauth.c

```diff
@@ -1078,6 +1081,11 @@ cockpit_session_launch (CockpitAuth *self,
 
       program_default = cockpit_ws_ssh_program;
     }
+  else if (g_strcmp0 (type, "tls-cert") == 0 &&
```

Comment:
There should be a massive TODO here. In particular this config option should probably be describing the CA cert we trust.

Reply:
See above, cockpit-ws isn't supposed to, nor is in a position to validate client cert chain.